charts/alfresco-search-enterprise/values.yaml

```diff
@@ -65,6 +65,12 @@ reindexing:
         # -- Key within the configmap holding the full url to connect to the
         # alfresco repository
         url: REPOSITORY_URL
+  # -- Helm hooks definition
+  policy:
+    # -- When the job will be executed
+    run: post-install
```

Comment:
`policy` doesn't look quite descriptive of what we are doing here.

`hook` as top level key and `name` for the hook event?